frontend/src/pages/documentation/Contribute.js

```diff
@@ -73,27 +72,27 @@ const Contribute = ({ history, ...props }) => {
         <React.Fragment>
           <p>
             OperatorHub.io is the front-end for the{' '}
-            <ExternalLink href={operatorsRepo} text={"Community Operators Repository"} indicator={false} />
-            This is an open source community project aiming to curate and collect Kubernetes Operators. Community
-            contributions live in:
+            <ExternalLink href={operatorsRepo} text="Community Operators Repository" indicator={false} />. This is an
+            open source community project aiming to curate and collect Kubernetes Operators. Community contributions
+            live in:
           </p>
           <p>
             <ExternalLink href={contributions} text={contributions} indicator={false} />
           </p>
           <p>
             If you have an Operator you would like to contribute to OperatorHub.io, feel free to create a pull request
             against the above directory. Please read the pull request{' '}
-            <ExternalLink href={operatorsRepoBeforePR} text="requirements" indicator={false} />
-            before creating one to ensure all required files are present.
+            <ExternalLink href={operatorsRepoBeforePR} text="requirements" indicator={false} /> before creating one to
+            ensure all required files are present.
           </p>
           <p>
             The easiest way to package your Operator for OperatorHub.io is to build it with the{' '}
             <ExternalLink href={operatorSdk} text="Operator SDK" indicator={false} />, however it is not a requirement.
             To publish your Operator, it must be built as a binary within a container, and that container must be hosted
-            on a publicly accessible{' '}
-            <ExternalLink href={operatorRegistry} text="registry" indicator={false} />. It should be accompanied by
-            some{' '} <ExternalLink href={operatorsRepoRequirements} text="metadata" indicator={false} /> that is
-            used for deploying the Operator using the{' '}
+            on a publicly accessible <ExternalLink href={operatorRegistry} text="registry" indicator={false} />. It
+            should be accompanied by some{' '}
```

Comment:
wouldn't it be nicer to use &nbsp; instead of {' '} to force space?

Reply:
the space will be removed when it's at the end of a line.

Reply:
oh, I wrote "& nbsp;" but it was replaced with space :D

Reply:
"&nbsp"

Reply:
I've used this method (`{' '}`) throughout so I'd say keep it consistent.